source/photo.html

```diff
@@ -36,16 +37,18 @@
           </ul>
         </div>
       </nav>
-      <img class="page-header__title page-header__title--main" src="img/title.svg" width="282" height="49" alt="Город Седона">
+      <div class="page-header__logo">
+        <img class="page-header__title page-header__title--main" src="img/title.svg" width="282" height="49" alt="Город Седона">
+      </div>
     </header>
     <main>
-      <div class="main-header">
+      <div class="main-header main-header-photo">
```

Comment:
```suggestion
      <div class="main-header main-header--photo">
```
Модификатор ведь? Класс `main-header-photo` указывает на то, что это блок.